src/components/MenuPanel/WalletManager/Details/Actions/Actions.js

```diff
@@ -12,21 +12,11 @@ function MenuPanelWalletManagerDetailsActions() {
         path='/wallets'
         label='Manage wallets'
       />
-      <MenuPanelActionsItem
-        icon='lock'
-        path='#'
-        label='Lock the app'
-      />
       <MenuPanelActionsItem
         icon='plus'
-        path='#'
+        path='/wallets/create'
         label='Add new wallet'
```

Comment:
Im not sure that 'Add new wallet' is only `create` action, but not `import` also.

Reply:
Yep, but it could not be both. :)